apps/portals/nf/src/config/synapseConfigs/datasets.ts

```diff
@@ -46,7 +46,7 @@ export const datasetCardConfiguration: CardConfiguration = {
       'visualizeDataOn',
     ],
     dataTypeIconNames: 'dataType',
-    downloadCartSynId: 'id',
+    useRowIdAndVersionForDownloadCart: true,
```

Comment:
Just use the rowId/version for adding to download cart, since it's a dataset collection.